### PR TITLE
feat: contribution guide

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes https://github.com/KILTprotocol/ticket/issues/1847

Adds the contribution

## How to test:

yarn start

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: docs/development/01-contribution.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+For the integrity of the protocol, requesting a feature requires approval from KILT governance. The request should follow the guidelines below, which are subject to change:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+
+ - Give your proposal a clear title
+ - Open an issue on the corresponding repository
+ - Provide a written outline of the feature request for discussion
+
+At the end of the discussion, the proposal needs the approval of the community, KILT Technical Committee, governance, or relevant project team (depending on the subject) to progress to the next stage. If approved, acceptable deliverables will be agreed with the proposer.
```
Comment (tjwelde) [inline docs/development/01-contribution.md]: So is this just for requesting features that KILT develops, or is it proposing a feature, that the proposer develops?

Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: I guess this could be both.

There could be a final closing remark here stating that the proposal or request must be taken by a team or done by the proposer.

File: docs/development/00-introduction.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+id: introduction
+title: Development
+---
+
+KILT protocol is an open-source project built upon the [substrate framework](https://docs.substrate.io/). The guide is to help builders and contributors find the resources needed to take action and work within the guideance of the active developers.
```
Comment (ntn-x2) [inline docs/development/00-introduction.md]:
```suggestion
KILT protocol is an open-source project built upon the [Substrate framework](https://docs.substrate.io/). The guide is to help builders and contributors find the resources needed to take action and work within the guidance of the active developers.
```

File: docs/development/01-contribution.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- provide feedback on proposals or ideas
+- discuss possible use-cases or feature requests
+- make suggestions for non-technical contributions, including events, writing, or business models
+- answer questions about the protocol, services and products.
+
+## Feature Requests
+
+For the integrity of the protocol, requesting a feature requires approval from KILT governance. The request should follow the guidelines below, which are subject to change:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals
```
Comment (ntn-x2) [inline docs/development/01-contribution.md]: Maybe?

```suggestion
 - Begin a discussion with the community to ensure that they agree that it adds value to the KILT protocol in a meaningful way, supporting its goals
```

File: docs/development/01-contribution.md (hunk 1 above)
Comment (tjwelde) [reply docs/development/01-contribution.md]: I like the split, but I think in the "Feature Proposal" should mention, that a proposal has to be made. because there is no proposal at that point.

File: docs/development/01-contribution.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+For the integrity of the protocol, requesting a feature requires approval from KILT governance. The request should follow the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds value to the KILT protocol in a meaningful way, supporting its goals
+ - Give your proposal a clear title
+ - Open an issue on the corresponding repository
+ - Provide a written outline of the feature request for discussion
+
+## Feature Proposals
+
+A proposal must be made before moving ahead, see above. At the end of the discussion, the proposal needs the approval of the community, KILT Technical Committee, governance, or relevant project team (depending on the subject) to progress to the next stage. If approved, acceptable deliverables will be agreed with the proposer, and the active developers or appropriate teams can start working on the proposal deliverables.
```
Comment (tjwelde) [inline docs/development/01-contribution.md]: I still think it is confusing and mixing things up. AFAICS, this section should just be about proposal, which the proposer will have to do and the previous section is just for requesting features, which kilt will have to do...
I think it needs a bit more thought on how to actually structure these sections.

File: docs/development/01-contribution.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+ - Open an issue on the corresponding repository
+ - Give your request a clear title
+ - Provide a written outline of the feature request for discussion
+
+After a discussion, a request may move further and requires implementation. If this is the case and needs implementation from a team that may require a treasure proposal: Check out the treasury proposals section.
+
+## Treasury Proposals
+
+A treasury proposal is a request to spend funds within the treasure pool. Usually, the proposals have altered or enhanced KILT to better the protocol in a meaningful way. The proposal should begin with a proposal with a clear title and a written outline of the idea and a discussion about implementation or deliverables as outlined above for feature requests.
```
Comment (tjwelde) [inline docs/development/01-contribution.md]:
```suggestion
A treasury proposal is a request to spend funds from the treasure pool. Usually, the proposals would alter or enhance KILT to better the protocol in a meaningful way. The proposal should begin with a clear title and a written outline of the idea and a discussion about implementation or deliverables as outlined above for feature requests.
```

File: docs/development/01-contribution.md (hunk 4 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: Updated

File: docs/development/00-introduction.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+id: introduction
+title: Development
+---
+
+KILT protocol is an open-source project built upon the [Substrate framework](https://docs.substrate.io/). The guide is to help builders and contributors find the resources needed to take action and work within the guidance of the active developers.
```
Comment (weichweich) [inline docs/development/00-introduction.md]: One sentence per line please. Better to review and better for git diffs.

```suggestion
KILT protocol is an open-source project built upon the [Substrate framework](https://docs.substrate.io/).
The guide is to help builders and contributors find the resources needed to take action and work within the guidance of the active developers.
```

File: docs/development/01-contribution.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT Protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
```
Comment (weichweich) [inline docs/development/01-contribution.md]:
```suggestion
As a decentralized network, KILT depends on the support of its community.
There are many ways to contribute to KILT Protocol and the products and services built on it. 

If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel.
The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
```

File: docs/development/01-contribution.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+ - Open an issue on the corresponding repository
+ - Give your request a clear title
+ - Provide a written outline of the feature request for discussion
+
+After discussion, if the community agrees that the change should be implemented, the proposer may also make a Treasury proposal to support the work. Check out the treasury proposals section.
+
+## Treasury Proposals
+
+A Treasury proposal is a request to receive funds from the Treasury pool. The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. The proposal should be for something that changes or adds to KILT in a meaningful way.
```
Comment (weichweich) [inline docs/development/01-contribution.md]:
```suggestion
A Treasury proposal is a request to receive funds from the Treasury pool.
The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests.
The proposal should be for something that changes or adds to KILT in a meaningful way. 
```

File: docs/development/01-contribution.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+A Treasury proposal is a request to receive funds from the Treasury pool. The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. The proposal should be for something that changes or adds to KILT in a meaningful way. 
+
+In addition you should:
+
+
+- Explain any deliverables that have already been achieved
+- Outline what needs to be done for the proposal to be completed
+
+The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT). If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance. See the [guide to creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
```
Comment (weichweich) [inline docs/development/01-contribution.md]:
```suggestion
The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT).
If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance.
See the [guide to creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
```

File: docs/development/00-introduction.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+---
+id: introduction
+title: Development
+---
+
+KILT protocol is an open-source project built upon the [Substrate framework](https://docs.substrate.io/).
+The guide is to help builders and contributors find the resources needed to take action and work within the guidance of the active developers.
+
+- Do you want to start contributing? Check out our [contribution guide](./01-contribution.md) to get started and helping out!
```
Comment (wischli) [inline docs/development/00-introduction.md]: The `helping out` feels off to me. Did Majella review this text already?

```suggestion
- Do you want to start contributing? Check out our [contribution guide](./01-contribution.md) to get started!
```

File: docs/development/01-contribution.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+## Bug Reports
+
+We try our best, but bugs are an everyday reality with all software and are bound to happen. We can't fix bugs we don't notice, so your input if you spot something gives us the best possibility to keep the project running smoothly and securely.
+
+If you are unsure that a bug is a bug, it is best to open an issue and report it anyway. The active developers will evaluate it and help to figure out the issue.
+
+It is helpful to check if a report has already been filed in the related project. Search the issues board for possible phrases that match the description of the bug. It's possible you may not find an issue, but it's better to file a duplicated bug than not report one.
+
+Once you begin reporting the bug, write a descriptive title so that if others find the same issue they can either add to your findings or know that the bug has already been reported. A bug report should be as detailed as possible, including steps to reproduce, screenshots, error reports, or code snippets. The more details you provide, the easier it is to fix the issue.
```
Comment (weichweich) [inline docs/development/01-contribution.md]:
```suggestion
We try our best, but bugs are an everyday reality with all software and are bound to happen.
We can't fix bugs we don't notice, so your input if you spot something gives us the best possibility to keep the project running smoothly and securely.

If you are unsure that a bug is a bug, it is best to open an issue and report it anyway.
The active developers will evaluate it and help to figure out the issue.

It is helpful to check if a report has already been filed in the related project.
Search the issues board for possible phrases that match the description of the bug.
It's possible you may not find an issue, but it's better to file a duplicated bug than not report one.

Once you begin reporting the bug, write a descriptive title so that if others find the same issue they can either add to your findings or know that the bug has already been reported.
A bug report should be as detailed as possible, including steps to reproduce, screenshots, error reports, or code snippets.
The more details you provide, the easier it is to fix the issue.
```

File: docs/development/01-contribution.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+
+## Pull Requests
+
+Pull Requests (PR) are an integral part of contributions to change KILT. GitHub itself has some [excellent documentation](https://help.github.com/articles/about-pull-requests/) on using the Pull Request feature. KILT uses the "[fork and pull](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models)" model described here, where contributors push changes to their personal fork and create pull requests to bring those changes into the source repository.
+
+Before starting a PR, it’s best to contact other active developers and discuss the proposed changes. Open an issue or directly contact some of the devs on [Discord](https://discord.gg/7uyfMXh6AT) to begin discussing and making the proposal. Once approved, contributors can open a PR for review. The PR will be reviewed and, if accepted, merged into the corresponding repository.
```
Comment (weichweich) [inline docs/development/01-contribution.md]:
```suggestion
Pull Requests (PR) are an integral part of contributions to change KILT.
GitHub itself has some [excellent documentation](https://help.github.com/articles/about-pull-requests/) on using the Pull Request feature.
KILT uses the "[fork and pull](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models)" model described here, where contributors push changes to their personal fork and create pull requests to bring those changes into the source repository.

Before starting a PR, it’s best to contact other active developers and discuss the proposed changes.
Open an issue or directly contact some of the devs on [Discord](https://discord.gg/7uyfMXh6AT) to begin discussing and making the proposal.
Once approved, contributors can open a PR for review.
The PR will be reviewed and, if accepted, merged into the corresponding repository.
```

File: docs/development/01-contribution.md (hunk 7 above)
Comment (wischli) [inline docs/development/01-contribution.md]: New sentence, new line.

```suggestion
If you are interested in contributing but unsure how to begin, we recommend to start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT). The developers there, which include the team that originally developed KILT as well as the wider KILT community, can:
```

File: docs/development/01-contribution.md [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT Protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
+
+- Provide feedback on proposals or ideas
+- Discuss possible use-cases or feature requests
+- Make suggestions for non-technical contributions, including events, writing, or business models
+- Answer questions about the protocol, services and products.
```
Comment (wischli) [inline docs/development/01-contribution.md]: I would add either periods or commas to each bullet point:

```suggestion
- Provide feedback on proposals or ideas.
- Discuss possible use-cases or feature requests.
- Make suggestions for non-technical contributions, including events, writing, or business models.
- Answer questions about the protocol, services and products.
```

File: docs/development/01-contribution.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+
+
+## Feature Requests 
+
+A feature request may be used to change KILT and its services by adding, removing or altering features within the scope of KILT Protocol.
```
Comment (wischli) [inline docs/development/01-contribution.md]:
```suggestion
A feature request may be used to change the KILT Protocol and its services by adding, removing or altering features within the scope the protocol.
```

File: docs/development/01-contribution.md [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+
+A feature request is a meaningful way to contribute from anyone following the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals.
```
Comment (wischli) [inline docs/development/01-contribution.md]:
```suggestion
 - Begin a discussion with the community to ensure that they agree that it adds to the KILT Protocol in a meaningful way, supporting its goals. 
```

File: docs/development/01-contribution.md [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+## Feature Requests 
+
+A feature request may be used to change KILT and its services by adding, removing or altering features within the scope of KILT Protocol.
+
+A feature request is a meaningful way to contribute from anyone following the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+ - Open an issue on the corresponding repository
+ - Give your request a clear title
+ - Provide a written outline of the feature request for discussion
```
Comment (wischli) [inline docs/development/01-contribution.md]: Again, would add periods to align with the other bullet point
```suggestion
 - Open an issue on the corresponding repository.
 - Give your request a clear title.
 - Provide a written outline of the feature request for discussion.
```

File: docs/development/01-contribution.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+
+After discussion, if the community agrees that the change should be implemented, the proposer may also make a Treasury proposal to support the work. Check out the treasury proposals section.
```
Comment (wischli) [inline docs/development/01-contribution.md]:
```suggestion
If the community agrees that the change should be implemented after discussion, the proposer may also make a Treasury proposal to support the work. Check out the [treasury proposals section](#treasury-proposals) below.
```

File: docs/development/01-contribution.md (hunk 9 above)
Comment (wischli) [inline docs/development/01-contribution.md]: Adding to Albi's request for new lines: I would use KILT Council instead of the technical committee and including `governance` to `community` does not make sense to me. Thus, I would rephrase it as `community and council` as the council decides on Treasury proposals, not the Technical Committee and not the normal governance (for now).
```suggestion
The proposal should then be discussed with the community and the KILT council using community channels such as [Discord](https://discord.gg/7uyfMXh6AT) or [Polkassembly](https://kilt.polkassembly.network/discussions).
If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance.
Please read through the [guide on creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
```

File: docs/development/01-contribution.md [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+
+## Treasury Proposals
+
+A Treasury proposal is a request to receive funds from the Treasury pool. The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. The proposal should be for something that changes or adds to KILT in a meaningful way. 
+
+In addition you should:
+
+
+- Explain any deliverables that have already been achieved
+- Outline what needs to be done for the proposal to be completed
```
Comment (wischli) [inline docs/development/01-contribution.md]: It's perio'clock

```suggestion

- Explain any deliverables that have already been achieved.
- Outline what needs to be done for the proposal to be completed.
```

File: docs/development/01-contribution.md (hunk 8 above)
Comment (wischli) [reply docs/development/01-contribution.md]: I am missing an important piece of information which states that proposals are funding completed contributions and that one should rather go for multiple ones instead of a big proposal. I believe the best place for this statement is in this paragraph, so I added it to my suggestion below.

```suggestion
... adds value to KILT in a meaningful way. 
In general, a Treasury proposal spent occurs after completing all outlined deliverables and not before.
Thus, it is recommended to rather open multiple subsequent milestone-based proposals instead of a large one to fund contributions.
```

File: docs/development/01-contribution.md [hunk 19]
```diff
@@ -0,0 +1,59 @@
...
+
+The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT). If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance. See the [guide to creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
+
+Governance approval for a Treasury spend will also  includes list of the deliverables as outlined in the proposal.
```
Comment (wischli) [inline docs/development/01-contribution.md]: What do you want to say with this statement? Governance approval is a boolean action. Of course, it can only occur, if all proposed deliverables are completed.

File: docs/development/01-contribution.md [hunk 20]
```diff
@@ -0,0 +1,59 @@
...
+
+## Bug Reports
+
+We try our best, but bugs are an everyday reality with all software and are bound to happen. We can't fix bugs we don't notice, so your input if you spot something gives us the best possibility to keep the project running smoothly and securely.
```
Comment (wischli) [inline docs/development/01-contribution.md]: I would like to improve the wording of this paragraph. Of course, the sentences need to be split into new lines but then the changes are harder to follow.

```suggestion
We try our best, but bugs are an everyday reality in software and are bound to happen. We cannot fix bugs which we do not notice. Thus, to give us the best possibility to keep the project running smoothly and securely, we encourage our community to inform us about any strange behaviour that you notice. Any input is valuable, so please do not hesitate to raise questions.
```

File: docs/development/01-contribution.md (hunk 11 above)
Comment (wischli) [inline docs/development/01-contribution.md]:
```suggestion
Once you begin reporting the bug, write a descriptive title. Thus, if others find the same issue, they can either add to your findings or know that the bug has already been reported. A bug report should be as detailed as possible, including steps to reproduce, screenshots, error reports, or code snippets. The more details you provide, the easier it is to fix the issue.
```

File: docs/development/01-contribution.md (hunk 12 above)
Comment (wischli) [inline docs/development/01-contribution.md]:
```suggestion
Before starting a PR, it is best to contact other active developers and discuss the proposed changes. Open an issue or directly contact some of the developers on [Discord](https://discord.gg/7uyfMXh6AT) to begin discussing and making the proposal. Once approved, contributors can open a PR for review. The PR will be reviewed and, if accepted, merged into the corresponding repository.
```

Comment (wischli) [reply docs/development/01-contribution.md]: We could also add mentioning of tips but should check with Gustav beforehand whether the phrasing is legally fine. Also, we might need an entire section on tips which would be out of scope of this PR. If you want, I can open a follow-up PR to add a section about tips.
```
Depending on the added value and scope of the contribution, it might be eligible for receiving a tip from the KILT Treasury.
```

File: docs/development/01-contribution.md [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+
+## Pull Requests
+
+Pull Requests (PR) are an integral part of contributions to change KILT. GitHub itself has some [excellent documentation](https://help.github.com/articles/about-pull-requests/) on using the Pull Request feature. KILT uses the "[fork and pull](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models)" model described here, where contributors push changes to their personal fork and create pull requests to bring those changes into the source repository.
+
+Before starting a PR, it’s best to contact other active developers and discuss the proposed changes. Open an issue or directly contact some of the devs on [Discord](https://discord.gg/7uyfMXh6AT) to begin discussing and making the proposal. Once approved, contributors can open a PR for review. The PR will be reviewed and, if accepted, merged into the corresponding repository.
+
+The following section is inspired by Rust langs [Bug Report](https://rustc-dev-guide.rust-lang.org/contributing.html) contribution guide.
```
Comment (wischli) [inline docs/development/01-contribution.md]:
```suggestion
The following section is inspired by the Rust Programming Language [Bug Report](https://rustc-dev-guide.rust-lang.org/contributing.html) contribution guide.
```

**Review by wischli — CHANGES_REQUESTED** [review]
I have a few language/wording remarks, as well as some content additions. 

Also, please migrate all sentences to a new line. This makes reviewing and editing way easier!

File: docs/development/00-introduction.md (hunk 10 above)
Comment (Dudleyneedham) [reply docs/development/00-introduction.md]: Yes, this is encouraging to participants.

File: docs/development/01-contribution.md (hunk 7 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: We have to use active developers when we refer the KILT community.

File: docs/development/01-contribution.md (hunk 13 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: Majella said no to this style and said at the end of all bullet points a fullstop is fine.

File: docs/development/01-contribution.md (hunk 17 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: I like the linkage, but the sentence reads better with `After discussion`

File: docs/development/01-contribution.md (hunk 8 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: Yes, I like the addition. I will add change.

Comment (Dudleyneedham) [reply docs/development/01-contribution.md]:
```suggestion
A Treasury proposal is a request to receive funds from the Treasury pool. 
The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. 
The proposal should be for something that changes or adds value to KILT in a meaningful way. 
In general, a Treasury proposal spent occurs after completing all outlined deliverables and not before.
Thus, it is recommended to rather open multiple subsequent milestone-based proposals instead of a large one to fund contributions.
```

File: docs/development/01-contribution.md (hunk 19 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: Yes, but that is only obvious if you know about the governance. It is a new concept and the readers potentially will not know about this. Therefore, we must be explicit in our speech and not let the reader knows.

File: docs/development/01-contribution.md (hunk 20 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: It feels repetitive. The two sentences are similar and the next sentence reinforces the issue.

File: docs/development/01-contribution.md (hunk 12 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: At the moment, I think we can leave the TIP out. Since we haven't had a chance for Gustav to check and its fine since we can include it later once we have more documentation relevant to the TIP otherwise it feels slapped in there.

File: docs/development/01-contribution.md (hunk 9 above)
Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: I think it is good to include different aspects of the community. The technical committee is a role within the community and can take part in discussions. 

Though I will include the polkassembly part.

Comment (Dudleyneedham) [reply docs/development/01-contribution.md]:
```suggestion
The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT) or [Polkassembly](https://kilt.polkassembly.network/discussions).
If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance.
See the [guide to creating a proposal](https://www.kilt.io/treasury/create-a-proposal/).
```

File: docs/development/01-contribution.md (hunk 16 above)
Comment (wischli) [reply docs/development/01-contribution.md]: @Dudleyneedham According to your statement above, you have to move the period to the last bullet point then.

File: docs/development/01-contribution.md (hunk 19 above)
Comment (wischli) [reply docs/development/01-contribution.md]: IMO, Governance mainly refers to Democracy (e.g. the entire community can vote in a public referendum). In the case of a Treasury proposal, voices can be raised but eventually the Council votes on it.

Comment (wischli) [reply docs/development/01-contribution.md]: In any case, the language is slightly off, isn't it?
```suggestion
Governance approval for a Treasury spend will also include a list of the deliverables as outlined in the proposal.
```

Comment (Dudleyneedham) [reply docs/development/01-contribution.md]: We can get rid of this statement.  :)

**Comment by ntn-x2** [comment]
@Dudleyneedham is this still relevant with the new docs structure? If so, can we do anything to bring this task to completion?

**Comment by wischli** [comment]
Closed in favor of #113
